src/components/spaces/cards/index.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
```

Comment:
Can you do the imports for consistency with how I've been ordering them? Like this:
```
import React from 'react'

import Icon from 'react-fa'

import * as navigationActions from 'gModules/navigation/actions'

import * as Space from 'gEngine/space'
import * as User from 'gEngine/user'

import {formatDescription, formatDate} from 'gComponents/spaces/shared'

import arrowsVisibleImage from '../../../assets/metric-icons/blue/arrows-visible.png'

import './style.css'
```
Ordered by sections like this, new lines separating sections:
    - React
    - Redux
    - NPM modules
    - Internal components
    - Internal selectors
    - Internal actions
    - Engine
    - Internal libs
    - servers
    - Images
    - Stylesheets